docs/sources/basics/exemplars/view-exemplars.md

```diff
@@ -0,0 +1,63 @@
++++
+title = "View exemplar data"
+description = "Exemplars"
+keywords = ["grafana", "concepts", "exemplars", "prometheus"]
+weight = 400
++++
+
+# View exemplar data
+
+When support for exemplar support is enabled for a Prometheus data source, you can view exemplar data either in the Explore view or from the Loki log details. 
+
+## In Explore
+
+Explore visualizes exemplar traces as highlighted stars alongside metrics data. For more information on how Explore visualizes trace data, refer to [tracing in Explore]({{< relref "../../explore/trace-integration.md" >}}).
```

Comment:
When you format a reference like this, where you say "refer to Title", match the capitalization of the link target.

Reply:
Fixed.